site/docs/contribute/logging.md

```diff
@@ -63,3 +63,5 @@ The fields in this section are typically generated by the logging libraries.
 - LOGGING_VERBOSITY - should be set to one of the levels described in the previous section.  
 - PRETTY_LOGGING - If true log entries are in human readable format.  If false, they are in json. Should only be true during individual development, since json is preferred to enable easy parsing by aggregator tools.
 
+## Logging of Structures
+Fybrik provides a helper function called `LogStructure` in pkg/logging/logging.go for writing Go structures in json format to the log.
```

Comment:
My point about documentation was that we need to provide some guidelines when to use `LogStructure` and when `log.Info` , `log.Trace` ....

Reply:
@roytman See https://fybrik.io/dev/contribute/logging/#log-entry-verbosity for when to use the different levels of logging.  LogStructure is when you want to print out a golang structure to be included in the log.  It handles the conversion to json.

Reply:
sorry, but I don't think that the explanation is clear.

Reply:
@roytman I rewrote it. Hopefully it's clearer now.